lib/tuples.gd

```diff
@@ -178,3 +178,43 @@ direct product elements families" );
 DeclareOperation( "DirectProductElement", [ IsList ]);
 DeclareOperation( "DirectProductElementNC",
     [ IsDirectProductElementFamily, IsList ]);
+
+
+#############################################################################
+##
+##
+##  <#GAPDoc Label="DirectProductFamily">
+##  <ManSection>
+##  <Func Name="DirectProductFamily" Arg='args'/>
+##
+##  <Description>
+##  <A>args</A> must be a dense list of <Ref Attr="CollectionsFamily"/>
+##  families, otherwise the function raises an error.
+##  <P/>
+##  <Ref Func="DirectProductFamily"/> returns <C>fam</C>, a collections
+##  family of <Ref Filt="IsDirectProductElement"/> objects.
+##  <P/>
+##  <C>fam</C> is the <Ref Filt="CollectionsFamily"/> of
+##  <Ref Filt="IsDirectProductElement"/> objects
+##  whose <C>i</C>-th component is in <C>ElementsFamily(args[i])</C>.
+##  <P/>
+##  Note that a collection in <C>fam</C> must thus not necessarily be a
+##  direct product.
+##  Also note that not all direct products in &GAP; are created via these
+##  families. For example the elements of a permutation group created via
```

Comment:
This requires the reader to know how to create permutation group "via DirectProduct", resp. to at least know this is possible. So perhaps like this:
```suggestion
##  families. For example if the arguments to <Ref Func="DirectProduct"/> are permutation groups,
```
cont'd below